src/graphics/shader.js

```diff
@@ -1,5 +1,7 @@
 /** @typedef {import('./graphics-device.js').GraphicsDevice} GraphicsDevice */
 
+import { EventHandler } from '../core/event-handler.js';
+
```

Comment:
While this isn't mentioned in the coding standards, typedefs come after imports. We should probably document module layout...

Reply:
done, thanks